enterprise_subsidy/apps/api/v1/views/subsidy.py

```diff
@@ -41,7 +48,13 @@ def __init__(self, can_redeem, content_price, unit, existing_transaction):  # py
 
 
 class SubsidyViewSet(
-    PermissionRequiredForListingMixin, mixins.RetrieveModelMixin, mixins.ListModelMixin, viewsets.GenericViewSet
+    PermissionRequiredForListingMixin,
+    mixins.CreateModelMixin,
+    mixins.DestroyModelMixin,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
```

Comment:
If this specific PR doesn't implement update(), then maybe we should remove mixins.UpdateModelMixin for now.